src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.trim().split(';');
+
+  const stylesObject = {};
+
+  stylesArray.forEach((style) => {
+    if (style.trim() === '') {
+      return;
+    }
+
+    const [property, value] = style.split(':');
+
+    stylesObject[property.trim()] = value.trim();
```

Comment:
Instead of using `forEach` and an external object, consider using `reduce` to build the styles object. This would align with the best practice of not modifying an existing object within an iteration method.